client/index.html

```diff
@@ -380,6 +379,18 @@ <h1 class="title">Settings</h1>
 								<textarea class="input" name="userStyles" id="user-specified-css-input" placeholder="You can override any style with CSS here"></textarea>
 							</div>
 						</div>
+
+						{{#unless public}}
+							<div class="session-list">
+								<h2>Sessions</h2>
+								
+								<h3>Current session</h3>
```

Comment:
I think the `session-list` id is not set where you want to set it, because we get:

<img width="485" alt="screen shot 2017-09-23 at 20 36 58" src="https://user-images.githubusercontent.com/113730/30778262-13c1aa8e-a09f-11e7-895d-debe0f50d202.png">

Probably want to hide both "Current session" and "Other sessions" when loading, then display them and hide the loading statement when it's available, no?

Reply:
Or I'm fine with no loading at all if you don't want to bother, it's pretty fast anyway. Just that at the moment it feels odd, I'm wondering if it's not leftovers from your previous version where you didn't have separate sections.